### PR TITLE
[18.0][MIG] server_action_mass_edit: Migration to 18.0

### PR DESCRIPTION
**Pull request by trisdoan**

### CODE REVIEW

File: server_action_mass_edit/models/ir_actions_server.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+        return {
+            "name": self.name,
+            "type": "ir.actions.act_window",
+            "res_model": "mass.editing.wizard",
+            "context": str(context),
+            "view_mode": "form",
+            "views": [[view_temp.id, "form"]],
+            "target": "new",
+        }
```
Comment (thienvh332) [inline server_action_mass_edit/models/ir_actions_server.py:72]: I don't see this function being used anywhere else.
Let's add a **`TODO`** to check and remove it in the next version.

Comment (trisdoan) [reply server_action_mass_edit/models/ir_actions_server.py:72]: Hi, great catch, thanks

If it's really not in use, it's better to remove it, imo.

It should target 17.0 and separate in another PR, dont you think ?

Comment (thienvh332) [reply server_action_mass_edit/models/ir_actions_server.py:72]: Ok for me

**Review by thienvh332 — APPROVED** [review]
LGTM

**Pull request closed without merging**